Introduction Draft1

mcommer opened the pull request:
First draft of introduction

Commit:
Introduction Draft1

stderr-enst reviewed (commented):
Danke Michael! Ich finde der Abschnitt passt schon sehr gut!
Ich war mir zuerst nicht sicher ob ich so viel Zeit mit dem eher synthetischen `bc` Beispiel verwenden würde, aber ich denke es ist eine gute Idee sich die Zeit zu nehmen und wirklich an einem Beispiel und im Vergleich zu sehen, wie sich die Performance durch kleine Änderungen **im workload** unterscheiden können.
Das ist Beispielhaft für jede Anwendung und ist hier ziemlich gut auf den Punkt gebracht.

Allerdings wäre es glaube ich gut, wenn wir zum Ende der Introduction zum Snowman Beispiel hinleiten. Ich würde also bei deinem `bc`-Beispiel bleiben aber dann weiter machen, so nach dem Motto "Wir machen jetzt nicht mit detaillierten Berechnungen weiter, sondern verwenden für den Rest des Kurses einen Platzhalter, der jeden eurer Use-Cases darstellen könnte: Ein Raytracer der eine Szene mit 3 Schneemännern berechnet."
Und dann können wir das erste Job Script schreiben, ein `time` for den raytracer werfen und haben auch einen Aufbau für alles Folgende.

Ich habe ein paare Kommentare direkt bei den entsprechenden Abschnitten hinterlassen.

Edit: 
Sorry bei den Zeilen-Bezogenen Kommentaren ist manchmal auch die rote Seite des Diffs dabei. Denke sie dir weg, darauf beziehe ich mich nicht :)

Commit:
Introduction draft version 2

Commit:
Introduction draft, version 2b

Commit:
Intro version, shortened, final draft, will still edit while merging with other episodes

Commit:
Changed awk example to bc with for loop, (hopefully) final intro draft

mcommer commented:
Replaced `awk` example by `bc` in order to stay consistent with the example. Also added short slurm job script at end of Intro episode for showcasing a first trial run.
